### ISSUE TITLE
Unknown custom element: <sweet-modal> 

### ISSUE THREAD

**Issue by packytagliaferro** [adeptoas/sweet-modal-vue#22]
For some reason this stopped working today. 

App.js:

```
import { SweetModal, SweetModalTab } from 'sweet-modal-vue'

Vue.component('SweetModal', SweetModal);

const app = new Vue({
    el: '#app',
});

```

php file:
```

<sweet-modal ref="wodModal">

      <wod-new></wod-new>

</sweet-modal>
```

**Comment by packytagliaferro** [comment]
I have also tried:

Vue.component(SweetModal);

and 

const app = new Vue({
    el: '#app',
     components:{
	SweetModal
      }
});

**Comment by bluefirex** [comment]
The last approach us the correct one. What's the exact error you get?

**Comment by Sopamo** [comment]
Seems like this is a bug introduced in 1.1.1. I downgraded to 1.1.0, that fixed the problem.

**Comment by packytagliaferro** [comment]
I just ended up using `Vue.component(SweetModal);` in my app.js file above my vue instance with my other components and then downgraded and it seemed to work. 

The error was:

`Unknown custom element: <sweet-modal> - did you register the component correctly? For recursive components, make sure to provide the "name" option.`

**Comment by dubcanada** [comment]
Getting this same error with the latest sweetmodal, was fine with 1.1.0

**Comment by Fleeck** [comment]
Same thing here. Everything worked before upgrading to 1.1.1; Downgrade fixes the problem. Probably there's something wrong with imports.

**Comment by Dmitry1987** [comment]
same here, 1.1.0 doesn't cause this error

**Comment by bluefirex** [comment]
Temporarily fixed in b449fac94731595f8afa7728877e8e018849c3da, released in [v1.1.2](https://github.com/adeptoas/sweet-modal-vue/releases/tag/v1.1.2)
